config/default.js

```diff
@@ -139,15 +154,17 @@ module.exports = {
         // For backwards-compatibility, allow images to be served from the old
         // dedicated domain for statics and user uploaded content:
         'https://addons.cdn.mozilla.net/',
```

Comment:
Another question about backwards-compatible settings that must be obsolete by now 😉

Reply:
That one we can definitely remove, the domain doesn't even work anymore, hasn't for a long time.